buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt

```diff
@@ -319,10 +319,6 @@ class PerformanceTestPlugin : Plugin<Project> {
                 module {
                     testSourceDirs = testSourceDirs + performanceTestSourceSet.groovy.srcDirs
                     testResourceDirs = testResourceDirs + performanceTestSourceSet.resources.srcDirs
-                    scopes["TEST"]!!["plus"]!!.apply {
```

Comment:
Is this required for Eclipse?

Reply:
This is the `IdeaModel`. It is not used in Eclipse import.

We also do not officially support importing the project in Eclipse. It can't fully work because we do not have the "one module per source set concept" in Eclipse.